src/pages/docs/administration/migrate-spaces-with-octoterra/index.md

```diff
@@ -0,0 +1,309 @@
+﻿---
+layout: src/layouts/Default.astro
+pubDate: 2099-01-01
+modDate: 2099-01-01
+title: Migrating spaces with octoterra
+description: How to migrate spaces using the octoterra tool
+navOrder: 100
+hideInThisSection: true
+navSearch: false
+navSitemap: false
+navMenu: false
+robots: noindex, follow
+---
+
+[Octoterra](github.com/OctopusSolutionsEngineering/OctopusTerraformExport/actions) exports Octopus projects, runbooks, and spaces to a Terraform module. Octoterra can be used to migrate resources between spaces and instances.
+
+## Limitations of octoterra and migrating projects between instances
+
+There are limitations that must be accounted for as part of a migration.
+
+### Sensitive values
+
+Octoterra reads the state of a space via the Octopus API and the API does not expose sensitive values. This means that octoterra can not export:
+
+* The value of sensitive variables associated with a project, tenant, or library variable set 
+* Credentials defined in feeds, accounts, or git credentials
+* The contents of a certificate
+* Sensitive values defined in steps
+
+Octoterra creates Terraform variables to allow the value of these fields to be defined when the module is applied.
+
+### Step templates
+
+The Octopus Terraform provider does not yet support defining step templates. Because step templates are assigned a unique ID in each space, it is not possible to export a project or runbook that references a step template across spaces.
+
+As a workaround, octoterra can detach step templates while exporting a project or runbook. Projects or runbooks with detached step templates can be recreated in a new space.
+
+### New step framework
+
+Some steps, such as the ECS deployment steps, rely on a new framework. Steps that use the new framework are not currently supported by the Terraform provider. These steps can not be exported by octoterra.
```

Comment:
I think it's worth mentioning which steps this includes as our customers wouldn't know and our team might not know. AFAIK ... 
- Amazon ECS Steps
- Bicep step 
- S3 step

Reply:
Note: I've confirmed the list in Slack.